jep/229/README.adoc

```diff
@@ -286,6 +279,29 @@ On balance this “pull” approach seems worse than the currently proposed “p
 * Theft of tokens prior to expiry by malicious maintainers is a possibility under either system;
   the window of opportunity would differ, as would the sort of audit trail produced.
 
+=== Release Drafter as a step in the release workflow
+
+Unfortunately it is not possible to run Release Drafter as a step in the release workflow,
```

Comment:
https://github.com/jenkinsci/core-changelog-generator has a changelog generator mode which takes the Release Drafter format. If needed, we can use it in the recommended flow inside Jenkins or elsewhere

Reply:
FTR https://github.com/jenkinsci/core-changelog-generator/pull/12 . Some patches are still needed to support all Release Drafter options, the patch was designed specifically for the Jenkins core

Reply:
Thanks. I think it is safest to stick with using official RD even if it needs to be done as a separate workflow at least for now. Can be revisited as we get real experience though. Will note your idea.